<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CraftDialog</class>
 <widget class="QDialog" name="CraftDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="dropMenu1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dropMenu2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="craftButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Craft</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <zorder>dropMenu2</zorder>
  <zorder>craftButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>dropMenu1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
